src/postprocess/band_structure.jl

```diff
@@ -31,10 +31,10 @@ function compute_bands(ham::Hamiltonian, kpoints, n_bands;
     select_eigenpairs_all_kblocks(band_data, 1:n_bands)
 end
 
-function plot_bands(ham, n_bands, kline_density, composition, εF)
+function plot_bands(ham, n_bands, kline_density, atoms, εF)
```

Comment:
Here I would not have the atoms at all and use the convenience function from above.